tck/README.md

```diff
@@ -41,7 +41,7 @@ Since the TCK is aimed at Reactive Stream implementers, looking into the sources
 In order to make mapping between test cases and Specification rules easier, each test case covering a specific
 Specification rule abides the following naming convention: `TYPE_spec###_DESC` where:
 
-* `TYPE` is one of: [#type-required](required), [#type-optional](optional), [#type-stochastic](stochastic) or [#type-untested](untested) which describe if this test is covering a Rule that MUST or SHOULD be implemented. The specific words are explained in detail below.
+* `TYPE` is one of: [required](#type-required), [optional](#type-optional), [stochastic](#type-stochastic) or [untested](#type-untested) which describe if this test is covering a Rule that MUST or SHOULD be implemented. The specific words are explained in detail below.
```

Comment:
LGTM, could you apply the same fix in https://github.com/reactive-streams/reactive-streams-jvm/blob/master/tck-flow/README.md ?

Reply:
ok :-)